credit/parser.py

```diff
@@ -710,6 +710,9 @@ def credit_main_parser(
             "train_batch_size" in conf["trainer"]
         ), "Training set batch size ('train_batch_size') is missing from onf['trainer']"
 
+        if "ensemble_size" not in conf["trainer"]:
+            conf["trainer"]["ensemble_size"] = 1
```

Comment:
Can you have an option is skip the ensemble routine? For those who do not use ensemble training, going through the ensemble code can cause a slow down.